src/behaviors/token_admin.rs

```
@@ -130,6 +140,20 @@ impl Behavior<Message> for TokenAdmin {
                 }
                 Ok(ControlFlow::Continue)
             }
+            TokenAdminQuery::DeployRequest(deploy_request) => {
```

completely ignores error handling for now